index.bs

```diff
@@ -34,6 +34,8 @@ spec:html; type:dfn;
 spec:url; type:dfn;
  text:origin
  for:/; text:url
+spec:streams; type:dfn
+ text:writable stream writer
```

Comment:
Non-blocking nit: it'd be good to export this instead of adding it here.

Reply:
https://github.com/whatwg/streams/pull/1304

Reply:
Done.